src/tracks/InstrumentTrack.cpp

```diff
@@ -43,6 +44,8 @@
 namespace lmms
 {
 
+	static const int BANK_NONE = -1;
```

Comment:
It might be better to make this publicly available in a namespace or a class.

Reply:
Done, moved to InstrumentTrack class